DogeNetWebClient/src/app/reducers/messages.reducer.ts

```diff
@@ -0,0 +1,69 @@
+import * as Actions from 'src/app/actions/messages.action';
+import { MessagesDetailsModel } from 'src/app/core/api/models';
+
+export interface MessagesState {
+  items: Dialogue[];
+}
+export interface Dialogue {
+  id: number;
+  messages: MessagesDetailsModel[];
+}
+
+const initialState: MessagesState = {
+  items: new Array<Dialogue>(),
+};
+
+export const messagesNode = 'messages';
+
+export const messagesReducer = (
+  state = initialState,
+  action: Actions.All
+): MessagesState => {
+  switch (action.type) {
+    case Actions.SET_NEW_DIALOGUE:
```

Comment:
Use new approach for creating reducers here

Reply:
done